backend/grpcplugin/serve.go

```diff
@@ -52,11 +53,17 @@ func Serve(opts ServeOpts) error {
 		opts.GRPCServer = plugin.DefaultGRPCServer
 	}
 
+	plugKeys := []string{}
+	for k := range pSet {
+		plugKeys = append(plugKeys, k)
+	}
+	log.DefaultLogger.Debug("Serving plugin", "plugins", plugKeys)
 	plugin.Serve(&plugin.ServeConfig{
 		HandshakeConfig:  handshake,
 		VersionedPlugins: versionedPlugins,
 		GRPCServer:       opts.GRPCServer,
 	})
+	log.DefaultLogger.Debug("Plugin server exited")
```

Comment:
If we use a global var for logging we might as well use global functions. 

```go
log.Debug("Plugin server exited")
```

Reply:
@bergquist Sure, but that's out of scope for this PR?